shared/chat/conversation/info-panel/notifications/index.js

```diff
@@ -33,6 +33,80 @@ const SaveStateComponent = ({saveState}) => {
   }
 }
 
+const unmutedNotificationPrefs = (props: Props) => [
+  <Checkbox
```

Comment:
didn't change anything here, just turned into array and added keys

(can we use fragments yet?)

Reply:
Yes but you have to use react.fragment and not the shortcut <>

Reply:
Done, thanks!